frontend/src/main/java/com/google/tripmeout/frontend/servlet/TripServlet.java

```diff
@@ -0,0 +1,58 @@
+package com.google.tripmeout.frontend.servlet;
+
+import com.google.gson.Gson;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import com.google.tripmeout.frontend.TripModel;
+import com.google.tripmeout.frontend.error.TripMeOutException;
+import com.google.tripmeout.frontend.error.TripNotFoundException;
+import com.google.tripmeout.frontend.storage.InMemoryTripModelStorage;
+import java.io.IOException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@Singleton
+public final class TripServlet extends HttpServlet {
+  // The Trip get and delete servlet
+  private static final long serialVersionUID = 1L;
```

Comment:
FYI, we shouldn't actually need `serialVersionUID`. That's a requirement for `Serializable` types, which we don't have here.

Reply:
https://javaee.github.io/javaee-spec/javadocs/javax/servlet/http/HttpServlet.html - HttpServlet is serializable because... well I have no idea... maybe they were supposed to be self deployable?